packages/enrichPrescriptions/src/statusUpdates.ts

```diff
@@ -109,20 +117,34 @@ export function applyStatusUpdates(searchsetBundle: Bundle, statusUpdates: Statu
 
     logger.info(`Applying updates for prescription ${prescriptionID}.`)
 
-    const prescriptionUpdate = statusUpdates.prescriptions.filter(p => p.prescriptionID === prescriptionID)[0]
+    const prescriptionUpdate = statusUpdates.prescriptions.filter((p) => p.prescriptionID === prescriptionID)[0]
     if (!prescriptionUpdate || !prescriptionUpdate.onboarded) {
       logger.info(`Supplier of prescription ${prescriptionID} not onboarded. Applying default updates.`)
-      medicationRequests?.forEach(medicationRequest =>
+      medicationRequests?.forEach((medicationRequest) => {
+        if (delayWithPharmacyStatus(medicationRequest)) {
+          logger.info(`Delaying 'With Pharamcy' status for prescription ${prescriptionID}`)
```

Comment:
Let's put the item id on this log too

Reply:
f50e933e95502ce9af85fc8306856be8fb4d627b